### PR TITLE
Start initial attempt from 0, not the end

### PR DESCRIPTION
**Pull request by eliperelman**
[bug 1455063](https://bugzilla.mozilla.org/show_bug.cgi?id=1455063)

This is part 1 of a 2-part fix. The other fix will be in taskcluster-tools once this lands.

### CODE REVIEW

File: src/fetch.js [hunk 1]
```diff
@@ -64,6 +64,6 @@ export default (url, opts = {}) => {
             setTimeout(() => attempt(n + 1), delay);
           }
         });
-    }(options.retries));
+    }(0));
```
Comment (djmitche) [inline src/fetch.js]: I see `n-1` above -- that will be -1**2, then 0**2.. I think you want to start at 1?

Comment (eliperelman) [reply src/fetch.js]: 🤔 yeah, you're probably right.

Comment (eliperelman) [reply src/fetch.js]: Revised.

**Comment by eliperelman** [comment]
Released in v7.0.0.

**Pull request merged**